### DISCUSSION TITLE
How to add related publications to project?

### DISCUSSION THREAD

**Discussion by JuergenFleiss** [alshedivat/al-folio#2405] [Q&A] [answered]
As per title, I am unable to figure out how to add related publications to a project and could not find any information in the documentation.

**Comment by george-gca** [comment, marked as answer]
We already have an example with this. Check our [project 1](https://alshedivat.github.io/al-folio/projects/1_project/) page.

**Reply by JuergenFleiss** [reply]
Hi, thanks for the answer. I looked trough the code and assume I can only display publications at the bottom if I cite the paper in the text?

**Reply by george-gca** [reply]
Yes, this is the supported solution. If you want to do it differently you have to implement it.

**Reply by JuergenFleiss** [reply]
Thanks a lot. Maybe I will have time to look into it.
